server/routes/utils.js

```diff
@@ -2,28 +2,28 @@ var jwt = require('jsonwebtoken');
 var config = require('../env');
 
 function verifyToken(req, res, next) {
-	var token = req.headers['x-access-token'];
+  var token = req.headers['x-access-token'];
 
-	if(!token) {
-		return res.status(403).send({
-			auth: false,
-			message: 'No token provided.'
-		});
-	}
+  if(!token) {
+    return res.status(403).send({
+      auth: false,
+      message: 'No token provided.',
+    });
+  }
 
-	jwt.verify(token, config.SECRET, function(err, decoded) {
-		if (err) {
-			return res.status(500).send({
-				auth: false,
-				message: 'Failed to authenticate token'
-			});
-		}
+  jwt.verify(token, config.SECRET, function(err, decoded) {
+    if (err) {
+      return res.status(500).send({
+        auth: false,
+        message: 'Failed to authenticate token',
```

Comment:
Adding trailing comma is because of this? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Reply:
I don't see any reasons from the link why should we use trailing commas, but main reason I like it is for diff. When a new line is added to the next line with comma, diff shows two lines were changed.

Reply:
yup agreed, for diff sounds reasonable. Will add trailing comma in the future code.